spec/lib/flipper/utilities/bulk_feature_checker_spec.rb

```diff
@@ -0,0 +1,146 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'flipper/utilities/bulk_feature_checker'
+
+RSpec.describe Flipper::Utilities::BulkFeatureChecker do
+  describe '.enabled_status' do
+    subject { Flipper::Utilities::BulkFeatureChecker.enabled_status(features) }
+
+    let(:features) { [] }
+    let(:empty_result) do
+      {
+        enabled: [],
+        disabled: [],
+        missing: []
+      }
+    end
+
+    context 'when empty array is provided for features' do
+      it 'returns an empty result' do
+        expect(subject).to eql(empty_result)
```

Comment:
Let's change these `eql`s to `match_array`s. This is sort of a general rule around array/hash matching - `eql` is strict in its matching (items in the array must be in the same order, or hashes must have their keys in the same order), whereas `match_array` doesn't care about ordering, just that the values are present & the same.

Reply:
Thanks, was unaware of that one! Makes a lot of sense. I've made the change. 👍